Framework/PythonInterface/core/CMakeLists.txt

```diff
@@ -4,141 +4,98 @@
 
 # It should not link to any other Mantid libraries.
 
-set(
-  SRC_FILES
-  src/ErrorHandling.cpp
-  src/ExtractWorkspace.cpp
```

Comment:
For my own understanding more than anything else. Why has ExtractWorkspace.cpp gone from this list?

Reply:
Having read further its because it is replaced with some improved handling of moving workspace objects into python so please ignore above comment.